src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -349,13 +367,36 @@ class CallOpcodesSpec extends WordSpec with Matchers with PropertyChecks {
         CallResult(op = CALL, context = context, gas = UInt256.MaxValue / 2)
       }
 
+      def callVarMemCost(config: EvmConfig): CallResult = {
+        val gasFailingBeforeEIP150Fix = 141072
```

Comment:
How did you arrive at this number? Maybe you could express it with function calls that calculate it?

Reply:
I have arrived at it by trial and error, as there is quite a narrow margin that `gasCap(state, gas, gExtra)` fails to execute because of OOG, but `gasCap(state, gas, gExtra + memCost)` executes normally.
Maybe a comment would be enough?

Reply:
Comment is fine